tests/e2e/specs/AskQuestion.js

```diff
@@ -1,46 +1,99 @@
 describe("Test navigation", () => {
+  beforeEach("checkDatabaseConnection", () => {
+    cy.intercept(
+      "POST",
+      "https://ama-tool-ea7e0-default-rtdb.europe-west1.firebasedatabase.app",
```

Comment:
Kann dies auch ein Regex oder wildcard sein? Also Situation: Wir ziehen die Firebase in einen offiziellen Firebase Bootcamp Account und die URL ändert sich leicht.